src/hotspot/cpu/x86/x86.ad

```diff
@@ -1650,6 +1657,9 @@ const bool Matcher::match_rule_supported_vector(int opcode, int vlen, BasicType
     case Op_RotateRightV:
     case Op_RotateLeftV:
     case Op_MacroLogicV:
+    case Op_VectorMaskLastTrue:
```

Comment:
But don't you support 128-/256-bit cases w/ AVX/AVX2 instructions?
This check effectively requires AVX512 as a baseline.